vyper/semantics/namespace.py

```diff
@@ -119,12 +120,15 @@ def override_global_namespace(ns):
         _namespace = tmp
 
 
-def validate_identifier(attr):
+def validate_identifier_with_namespace(attr):
```

Comment:
since this doesn't actually validate the identifier, maybe a better name would be `validate_namespace_availability`